bliss/simulator/decoder.py

```diff
@@ -91,6 +92,22 @@ def source_renderers(self):
             SourceType.GALAXY: self.render_galaxy,
         }
 
+    def pixel_shift(self):
+        """Generate random pixel shift and corresponding WCS list to undo shifts."""
+        shift = np.random.uniform(-2, 2, (self.n_bands, 2))
+        wcs_base = WCS()
+        base = np.array([5.0, 5.0])
+        wcs_base.wcs.crpix = base
+        wcs = []
+        for i in range(self.n_bands):
+            if i == 2:
```

Comment:
I suppose we should move the reference band to the config file rather than hardcoding it. I'm not sure where in the config to put it though. There's already a reference band for color in the config for the prior, but it's awkward to use that setting here.

Reply:
`-2` and `2` probably are survey-specific too. Should we put them in the config file too, so we can vary them for different surveys?

Reply:
I was thinking that about the reference band but I also just realized that it doesn't really matter which band is the reference band in this scenario - it can be arbitrary. For example, we could treat the band indexed with '0' to be the reference band for simplicity.

-2 and 2 as the amount of jitter could probably be placed in the config as well - we'd have to place that within the surveys config.

Reply:
I agree, as long the reference band used to generate the simulated data is the same one we use as the reference band at prediction time

Reply:
That's not the case currently - I think the fix is as simple as changing the`ref_band`within `predict()` (inside of `bliss.predict.py` when we call `align()` to be 0. (Referring to L116 and L121).

Reply:
We just used the `cfg.prior.reference_band` for simplicity and to avoid having multiple things called 'reference band.' But I think it does make sense to override a `cfg.predict.reference_band` instead.